src/Microsoft.ML.Transforms/StatefulFilterTransform.cs

```diff
@@ -178,13 +179,13 @@ public Cursor(StatefulFilterTransform<TSrc, TDst, TState> parent, RowCursor<TSrc
                 appendedDataView.SetCurrentRowObject(_dst);
 
                 Func<int, bool> appendedPredicate =
-                    col =>
-                    {
-                        col = _parent._bindings.AddedColumnIndices[col];
-                        return predicate(col);
-                    };
+                col =>
+                {
+                    col = _parent._bindings.AddedColumnIndices[col];
+                    return predicate(col);
+                };
 
-                _appendedRow = appendedDataView.GetRowCursor(appendedPredicate);
+                _appendedRow = appendedDataView.GetRowCursorForAllColumns();
```

Comment:
>GetRowCursorForAllColumns [](start = 48, length = 25)

it doesn't look the same as before.
And why we even need that appendedPredicate one line above?

Reply:
Should go. removed it, than added it back for debugging.. thanks. 

---
In reply to: [247208022](https://github.com/dotnet/machinelearning/pull/1975#discussion_r247208022) [](ancestors = 247208022)